api-gen/rendering/BUILD.bazel

```diff
@@ -0,0 +1,38 @@
+load("//tools:defaults.bzl", "ts_library")
+load("@build_bazel_rules_nodejs//:index.bzl", "nodejs_binary")
+
+package(default_visibility = ["//api-gen:__subpackages__"])
+
+ts_library(
+    name = "render_api_to_html_lib",
+    srcs = glob(["**/*.ts"]),
+    devmode_module = "commonjs",
+    deps = [
+        "@npm//@bazel/runfiles",
+        "@npm//@types/marked",
+        "@npm//@types/node",
+        "@npm//@types/nunjucks",
+        "@npm//marked",
+        "@npm//nunjucks",
+    ],
+)
+
+# nunjucks templates for generating API reference docs.
+filegroup(
+    name = "templates",
+    srcs = glob(["templates/*.njk"]),
```

Comment:
Just to capture the thoughts here. I know we talked about potentially using something else for templates. Any strong reason on why we use nunjucks? is it for re-using of dgeni templates?